packages/jest-cli/src/TestRunner.js

```diff
@@ -209,54 +198,43 @@ class TestRunner {
       const testResult = buildFailureTestResult(testPath, error);
       testResult.failureMessage = formatExecError(testResult, config, testPath);
       addResult(aggregatedResults, testResult);
-      this._dispatcher.onTestResult(
-        config,
-        testResult,
-        aggregatedResults,
-      );
+      this._dispatcher.onTestResult(config, testResult, aggregatedResults);
     };
 
     // Run in band if we only have one test or one worker available.
     // If we are confident from previous runs that the tests will finish quickly
     // we also run in band to reduce the overhead of spawning workers.
-    const shouldRunInBand = () => (
+    const shouldRunInBand = () =>
       this._options.maxWorkers <= 1 ||
       testPaths.length <= 1 ||
-      (
-        testPaths.length <= 20 &&
-        timings.length > 0 && timings.every(timing => timing < SLOW_TEST_TIME)
-      )
-    );
+      testPaths.length <= 20 &&
+        timings.length > 0 &&
+        timings.every(timing => timing < SLOW_TEST_TIME);
```

Comment:
Looks like we'll need to check `||` and `&&` more closely now (but this is actually what I'm used to)